api/open_saves.proto

```diff
@@ -383,6 +390,23 @@ message QueryFilter {
   Property value = 3;
 }
 
+// SortOrder is a way to order records returned by QueryRecords.
+message SortOrder {
+  // Direction is the ways to sort a list of records returned by querying.
+  enum Direction {
+    // Sort by ascending order.
+    ASC = 0;
+    // Sort by descending order.
+    DESC = 1;
+  }
+
+  // Property name to order by.
+  string property_name = 1;
```

Comment:
not a request but will we allow sorting by system properties (like updated_at or blob_size)? In that case using oneof might make sense here (although we only have property_name right now).